web/src/main/java/com/futurewei/alcor/web/entity/NetworkVlanRange.java

```diff
@@ -0,0 +1,80 @@
+package com.futurewei.alcor.web.entity;
+
+import com.futurewei.alcor.common.enumClass.NetworkTypeEnum;
+import com.futurewei.alcor.common.exception.KeyInvalidException;
+import com.futurewei.alcor.common.exception.VlanKeyAllocNotFoundException;
+import com.futurewei.alcor.web.allocator.VlanKeyAllocator;
+import lombok.Data;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+public class NetworkVlanRange {
+
+    private String id;
+    private String segmentId;
```

Comment:
Should a VlanRange be associated with a segment id?

Reply:
I see it in the resource that "If there are different tenants and different private networks are created, in order to isolate packets on the data network, you need to specify segmentid when creating the private network"

Reply:
yes but NetworkVlanRange is about the range of Vlan Segmentation Id, right? Let us discuss on these.

Reply:
I got the point. Actually the Range entity does not need segment Id, but network type entity does